docs/userGuide/plugins/filterTags.mbdf

```diff
@@ -1,7 +1,9 @@
-#### `filterTags`: Toggling alternative contents in a page
+## Filter tags
```

Comment:
`Filter tags` is a new term that is not mentioned anywhere else and not a standard term either.

Perhaps `Tags`?